src/lib/components/export-history.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import ToggleButton from '$lib/components/toggle-button.svelte';
+  import { fetchRawEvents } from '$lib/services/events-service';
+  import { faDownload } from '@fortawesome/free-solid-svg-icons';
+
+  const exportHistory = async () => {
+    const events = await fetchRawEvents({
+      namespace,
+      executionId: workflowId,
+      runId: runId,
+    });
+
+    const content = JSON.stringify(events, null, 2);
+    download(content, `${runId}/events.json`, 'text/plain');
+
+    function download(content: string, fileName: string, contentType: string) {
```

Comment:
Does the data URL method we were using not work?

Reply:
as we are working with potentially 10s of MBs, Blob + URL.createObjectURL seemed more appropriate instead of bloating the html with the data url

plus that's less custom logic and just using existing api

Reply:
switching to a dynamic <a> also allows to delay loading of 10s of MBs until that's needed

Reply:
Sold.